src/first_app.md

```diff
@@ -2,6 +2,10 @@
 
 For our first app, let's create something original: A simple counter app.
 
+![App screenshot dark](img/screenshots/simple-dark.png)
+
+In this app we will have a counter which can be increased and decreased by pressing the relevant buttons.
```

Comment:
I think relevant isn't the word for it. I'd rather use "corresponding button". Also increase and decrease seems to be less common than increment and decrement and to keep the book consistent I'd prefer in-/decrement.